components/GridToolBox/index.js

```diff
@@ -136,10 +136,19 @@ class GridToolBox extends Component {
             onChange(filterElement.name, obj.value);
         }
 
+        function onSelectCustomSearch(value) {
+            filterElement.field = value;
+            onChange('isExternalUser', value === 'userName' ? 0 : 1);
```

Comment:
This should not be hardcoded. GridToolbox have nothing to do with users or external users

Reply:
Fixed. There is MR in ut-user - https://git.softwaregroup.com/ut5/ut-user/-/merge_requests/2278